test/tests.js

```diff
@@ -1756,6 +1756,7 @@ user.getUserIdentities is not a function,\n`;
             baz: 'bar',
         };
 
+        debugger;
```

Comment:
noobie question, but wouldn't this pause the process of the test running and should be removed? I might be wrong

Reply:
Yes this is a mistake.  I left it here.  We should never have `debugger` in live code.